R/smc_post_processing_functions.R

```diff
@@ -429,6 +431,7 @@ compute_posterior_intervals_rho <- function(output, nmc, burnin, verbose=FALSE)
 #'
 compute_rho_consensus <- function(output, nmc, burnin, C, type, verbose=FALSE) {
   n_items <- dim(output)[2]
+  N <- dim(output)[1]
```

Comment:
N is the input variable nmc.
When using this function to analyse SMC, I set nmc = N.

Reply:
OK, so I'll change this line to `N <- nmc`. A bit redundant, but best for now IMO.